src/frontend/hooks/useJaNeiSpmFelt.tsx

```diff
@@ -30,23 +49,23 @@ const useJaNeiSpmFelt = (
                 ? ok(felt)
                 : feil(felt, <SpråkTekst id={språkTekstIdForFeil} />);
         },
-        skalFeltetVises: (avhengigheter: Avhengigheter) => {
+        skalFeltetVises: (avhengigheter: { [key: string]: FeltGruppe }) => {
+            if (!avhengigheter) return harBlittVist;
+            if (harBlittVist) {
```

Comment:
Denne må vi vel flytte ned igjen under spesialtilfellet for borPåRegistrertAdresse. Nå skjules jo ikke feltet hvis man endrer fra ja til nei på den.

Reply:
Ah, ja! Bra catch!

Reply:
Done!